ui/src/views/compute/DeployVM.vue

```diff
@@ -1442,6 +1444,9 @@ export default {
         if (template) {
           var size = template.size / (1024 * 1024 * 1024) || 0 // bytes to GB
           this.dataPreFill.minrootdisksize = Math.ceil(size)
+          this.defaultBootType = this.template?.details?.UEFI ? 'UEFI' : ''
+          this.fetchBootModes(this.defaultBootType)
+          this.defaultBootMode = this.template?.details?.UEFI
```

Comment:
Why this is declared twice?

Reply:
The boot type is determined if the `UEFI` key is present in the template details
the boot mode is the value of the key if present